packages/core/src/components/forms/_control-group.scss

```diff
@@ -100,20 +100,19 @@ Styleguide components.forms.control-group
       border-radius: $pt-border-radius;
     }
 
-    [readonly],
-    :disabled,
-    .pt-disabled {
-      z-index: index($control-group-stack, "input-disabled");
-      border-radius: 0;
```

Comment:
the whole block wasn't actually doing anything without the `&` before the attribs...